[Merge-Queue] Support multiple reviewers names

JonWBedard opened the pull request:
#### aa3111984a78c6d43f0d1a6f2a6723ddca299dde
<pre>
[Merge-Queue] Support multiple reviewers names
<a href="https://bugs.webkit.org/show_bug.cgi?id=238095">https://bugs.webkit.org/show_bug.cgi?id=238095</a>
&lt;rdar://problem/90503503 &gt;

Reviewed by Aakash Jain.

* Tools/CISupport/ews-build/steps.py:
(ApplyPatch.start): Only apply the first reviewer name.
(ValidateCommiterAndReviewer.start): Support a list of reviewers.
</pre>

aj062 commented on Tools/CISupport/ews-build/steps.py:
```diff
@@ -722,9 +722,9 @@ def start(self):
-        reviewer_name = self.getProperty('reviewer_full_name', '')
-        if reviewer_name:
-            self.command.extend(['--reviewer', reviewer_name])
+        reviewer_names = self.getProperty('reviewer_full_names', [])
```
reviewer_names => reviewers_names  (i was wondering why would a single reviewer have multiple names)

aj062 commented on Tools/CISupport/ews-build/steps.py:
```diff
@@ -1516,7 +1516,7 @@ def start(self):
             return None
 
-        self.setProperty('reviewer_full_name', self.full_name_from_email(reviewer))
+        self.setProperty('reviewer_full_names', [self.full_name_from_email(reviewer)])
```
Who is setting the 'reviewer' property? Seems like it is being set only for patch workflow, and currently we can't have multiple reviewers in that. It might be better to have this patch after or along-with the patch when we actually support having multiple reviewers in 'reviewer' property (for git workflow), but I am ok landing it before-hand as well if you prefer so.

aj062 replied:
Also, reviewer_full_names => reviewers_full_names

aj062 commented on Tools/CISupport/ews-build/steps.py:
```diff
@@ -722,9 +722,9 @@ def start(self):
-        if reviewer_name:
-            self.command.extend(['--reviewer', reviewer_name])
+        reviewer_names = self.getProperty('reviewer_full_names', [])
+        if reviewer_names:
```
Ditto

aj062 commented on Tools/ChangeLog:
```diff
@@ -1,3 +1,15 @@
+2022-03-18  Jonathan Bedard  <jbedard@apple.com>
+
+        [Merge-Queue] Support multiple reviewer names
```
Support multiple reviewer names => Support multiple reviewers names

JonWBedard committed:
[Merge-Queue] Support multiple reviewers names
https://bugs.webkit.org/show_bug.cgi?id=238095
<rdar://problem/90503503>

Reviewed by Aakash Jain.

* Tools/CISupport/ews-build/steps.py:
(ApplyPatch.start): Only apply the first reviewer name.
(ValidateCommiterAndReviewer.start): Support a list of reviewers.
